.gitignore

```diff
@@ -1,5 +1,4 @@
 *.log
-*.pyc
-env
```

Comment:
How come these are being removed?

Reply:
I noticed you purged the python hence the .pyc

but the `env`?

Reply:
I thought I would remove all reference to the python elements so no longer relevant in the gitignore